src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeFetchers.scala

```diff
@@ -591,7 +591,7 @@ trait BlocksDataFetchers {
     override val fetchData =
       Kleisli(
         blocks => {
-          logger.info("Fetching ballots in levels {}", blocks.head.data.header.level to blocks.last.data.header.level)
+          logger.info("Fetching ballots")
```

Comment:
It was nice to have this info.

Reply:
ah, yes, that's an issue which needed to be fixed, and I fixed just for testing - it happens that we have sometimes  empty list of blocks passed here, so `blocks.head` crashed the Lorre. 
Filtering of blocks was done to avoid making queries when there are no proposals - they would be empty.

So should I make another PR with this fix, or just keep it there?

Reply:
Whichever is quicker. :)

Reply:
hopefully we can merge it today :)